node/cloud_functions/basicBot/basicBot.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Author: Brett Reinhard
```

Comment:
Thanks for contributing Brett! A few things about the file header:

## Author comment

We don't put the author or date inside the code comments as these pieces of information are in git and the commit. Your contribution will link to your GitHub profile in the commit message and the file histories.

You also include the author in `package.json`, so we don't need to duplicate it here.

Use this header comment instead:

```js
/**
 * A basic Chat Bot setup, to be used with Google Cloud Functions.
 */
```

## License

In addition, please add a header like this as well like our other samples:
```

/**
 * Copyright 2018 Google Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```